integration/exec/main_test.go

```diff
@@ -0,0 +1,33 @@
+package exec
```

Comment:
`exec` is not a component (see https://github.com/moby/moby/tree/master/api/server/router). The component is `container` so this should go in `integration/container`.

Reply:
ok, I can move this to container.